### PR TITLE
Split rendering of sticks into separete js-file

### LINKED ISSUE #25: hidden stick display reappears
hidden stick display reappears when you press shortcut "S", "X" or "G" ... :(

### PR DESCRIPTION
**Pull request by wolkesson**
The main purpose is to clean up the code. But also fixes #25. This might conflict with #187.

### CODE REVIEW

File: js/grapher.js [hunk 1]
```diff
@@ -250,252 +239,26 @@ function FlightLogGrapher(flightLog, graphConfig, canvas, craftCanvas, analyserC
...
-    function drawFrameLabel(frameIndex, timeMsec)
-    {
-        var 
-            extentFrameNumber, extentFrameTime;
 
+    function drawFrameLabel(frameIndex, timeMsec) {
         canvasContext.font = drawingParams.fontSizeFrameLabel + "pt " + DEFAULT_FONT_FACE;
         canvasContext.fillStyle = "rgba(255,255,255,0.65)";
-
+
```
Comment (mikeller) [inline js/grapher.js]: Extra space.

**Comment by McGiverGim** [comment]
Great cleanup!! Any PR in this sense is a great addition.

About https://github.com/betaflight/blackbox-log-viewer/pull/187 I can redo/rebase it after. I can fight again with the mess of variables and options ;)

**Comment by wolkesson** [comment]
@McGiverGim Sorry for messing up your PR! Just shout if you need help. :)

**Comment by McGiverGim** [comment]
Yeah, don't worry. @wolkesson Please fix the review (the spaces left) as soon as possible to merge this, in this way I will work to adapt the other.

**Comment by wolkesson** [comment]
Strange seems VS code doesn't allow me to add them. If I press tab or spacebar to put spaces up to the indent level, they disappear when I leave the row. Must be setting somewhere... I've added them in notepad...

**Comment by wolkesson** [comment]
It's probably not a settings issue. The original spacing makes no sense, since in the same function it sometime has no indentation and sometimes "appropriate". VS Code tried to normalize this to "appropriate". Might be a good idea to "squash merge" this now.

**Comment by mikeller** [comment]
Yeah, I know that the existing spacing is borked in this project. It might be a good idea to open a separate pull request to fix it - it's way easier to review these changes if there are no changes in functionality mixed in.

**Comment by wolkesson** [comment]
Wow, this broke video export!!! Sorry, I completely forgot to check that! Submitted a fix #191!

**Pull request merged**